app/models/resource.py

```diff
@@ -12,16 +14,40 @@ class Resource(db.Model):
     reviews = db.relationship('ResourceReview', backref='resource',
                               lazy='dynamic')
 
+    def __init__(self, name, description, website, address):
+        self.name = name
+        self.description = description
+        self.website = website
+        self.address = address
+
     def __repr__(self):
         return '<Resource \'%s\'>' % self.name
 
+    @staticmethod
+    def generate_fake(count=10):
+        """Generate count fake Resources for testing."""
+        from faker import Faker
+        from random import choice
+
+        fake = Faker()
+
+        for i in range(count):
+            r = Resource(
+                name=fake.name(),
+                description=fake.text(),
+                website=fake.url(),
+                address=choice(Address.query.all())
```

Comment:
same thing as the zip code situation above, store this as a variable before the for loop